docs/install.md

```diff
@@ -174,15 +174,138 @@ dd if=xcp-ng-8.2.0-netinstall.iso of=/dev/sdX bs=8M oflag=direct
 
 Everything else is like the [regular install](install.md#start-the-host), except that it will not offer to install from local media, only from distant ones.
 
+## Network boot (PXE)
+
+### Requirements
+
+To get XCP-ng installed from PXE, you need:
+
+* DHCP and TFTP configured servers
+* Any NFS, FTP or HTTP server (your choice) to host XCP-ng installation files
+* A PXE-boot compatible network card on your host
+
+:::tip
+PXE boot doesn't support tagged VLAN networks! Be sure to boot on a untagged network!
+:::
+
+### TFTP server configuration
+
+1. In your TFTP root directory (eg `/tftp`), create a folder named `xcp-ng`.
+2. Copy the `mboot.c32` and `pxelinux.0` files from the installation media to the TFTP root directory.
+3. From the XCP-ng installation media, copy the files `install.img` (from the root directory), `vmlinuz`, and `xen.gz` (from the /boot directory) to the new `xcp-ng` directory on the TFTP server.
+4. In the TFTP root directory, create a folder called `pxelinux.cfg`
+5. In the pxelinux.cfg directory, create your configuration file called `default`.
+
+The file itself will contain the way to install XCP-ng: manually (with answer to provide on the host console/screen) or fully automated (see [Automated install](install.md#automatedinstall) below).
+
+Here is an example of a manual installation:
+
+```
+default xcp-ng
+label xcp-ng
+    kernel mboot.c32
+    append xcp-ng/xen.gz dom0_max_vcpus=2
```

Comment:
The multiple line breaks after the append line make it harder to understand that it's all one single entry. See https://updates.xcp-ng.org/netinstall/8.2/boot/isolinux/isolinux.cfg

By the way, are such line breaks supported without adding `\`?